app.js

```diff
@@ -2,4 +2,8 @@ require("babel-register")({
   presets: ["es2015"],
 });
 
-require("./backend/app.js");
+// Run backend with interval cache updates.
```

Comment:
also moved this code to trigger the cache updates to the top level app.js, so it doesn't get called on integration tests